.eslintrc.json

```diff
@@ -33,5 +33,13 @@
         ]
       }
     ]
-  }
+  },
+  "overrides": [
```

Comment:
I used overrides here, but I guess we could ignore package.json altogether too? Not sure how y'all feel about it.